packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -215,18 +215,40 @@ export class DotnetPluginImpl implements PluginImpl {
 
   public async postLocalDebug(ctx: PluginContext): Promise<TeamsFxResult> {
     const appSettingsPath = path.join(ctx.root, PathInfo.appSettingDevelopment);
+    let appSettings: string;
     if (
       !(await runWithErrorCatchAndThrow(
         new FileIOError(appSettingsPath),
         async () => await fs.pathExists(appSettingsPath)
       ))
     ) {
-      return ok(undefined);
+      // if appsetting file not exist, generate a new one
+      // TODO(qidon): load content from resource file or template
+      appSettings =
+        '\
+{\r\n\
```

Comment:
NIT: Prefer os.EOL.

Reply:
I believe we only support VS(windows) now, so this quick fix just use `\r\n`.
For further improvement considering share the same content with template.